### PR TITLE
search page and results

### LINKED ISSUE #26: Search
Use what already exists
Search data in #16

### PR DESCRIPTION
**Pull request by dlouise64**
merge after #41 
closes #26 
# What has changed and why?

# How to test this change

- [ ] has automated tests
- [x] at least one a reviewer ran the code
- [ ] tested in small screen
- [ ] i18n
- [ ] tested in screen reader/contrast <-- remove this if no UX change

### CODE REVIEW

File: src/components/Service/Service.js [hunk 1]
```diff
@@ -1,23 +1,66 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
 import '../../styles/Service.css';
-import ServiceDetail from './ServiceDetail';
-import ServiceContactDetail from './ServiceContactDetail';
-
+import fontawesome from '@fortawesome/fontawesome';
+// import FontAwesomeIcon from '@fortawesome/react-fontawesome';
```
Comment (Br3nda) [inline src/components/Service/Service.js]: can we remove these commented things?

**Comment by Br3nda** [comment]
Is this on this icon?

image

Nothing happens in firefox

**Comment by Br3nda** [comment]
I turned on the codeclimate comments instead of the hound ones.

**Comment by dlouise64** [comment]
@Br3nda search is now broken, I need to work on this.

**Comment by Br3nda** [comment]
git conflicts here.

**Comment by dlouise64** [comment]
@Br3nda conflicts fixed can I merge? I need to merge this one before the map feature

**Pull request merged**